app/js/services/page-provider.js

```diff
@@ -20,6 +20,11 @@ angular.module('app.services')
 			'list': 'partials/actions/organization/organization-list.html',
 			'add': 'partials/actions/event/event-add.html'
 		},
+		'positions': {
+			'list': 'partials/actions/position/position-list.html',
+			'edit':'partials/actions/position/position-edit.html',
```

Comment:
prolly should remove `position-edit` and `position-add` since you're not adding the html files / controller in this PR